modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -32,10 +45,12 @@ spec:
     node-role.kubernetes.io/worker: ""
   ptpEventConfig:
     enableEventPublisher: true <1>
-    transportHost: amqp://<instance_name>.<namespace>.svc.cluster.local <2>
+    transportHost: <transport_host> <2>
 ----
 <1> Set `enableEventPublisher` to `true` to enable PTP fast event notifications.
-<2> Set `transportHost` to the AMQ router that you configured where `<instance_name>` and `<namespace>` correspond to the AMQ Interconnect router instance name and namespace, for example, `amqp://amq-interconnect.amq-interconnect.svc.cluster.local`
+<2> Set `transportHost` to the HTTP Transport or AMQ router that you have configured.
+For example, for an HTTP Transport, set `http://ptp-event-publisher-service-compute-1-example.openshift-ptp.svc.cluster.local:9043`.
```

Comment:
Use the following http transport example, so that it is clear the node name is used.
http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043